### PR TITLE
WD-4924 Add the high-performance computing guide to HPC

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done

- Add a new strip for high-performance computing guide to HPC

## QA

- Open the demo
- Go to /hpc/use-cases
- Check that the new section matches the [copy doc](https://docs.google.com/document/d/1yG6svTYbKBgdGih5ASsWbRDWAMPcqIJVfpQ5LGoxRCE/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4924

### CODE REVIEW

**Comment by anthonydillon** [comment]
@lyubomir-popov mind taking a look at this one please?

**Comment by anthonydillon** [comment]
Demo running at https://ubuntu-com-13001.demos.haus/hpc/use-cases

File: templates/hpc/use-cases.html [hunk 1]
```diff
@@ -102,7 +102,31 @@ <h2 class="p-logo-section__title u-align--center">ADVANCING HIGH-PERFORMANCE WOR
       </div>
     </div>
   </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-8">
```
Comment (lyubomir-popov) [inline templates/hpc/use-cases.html:109]:
```suggestion
    <div class="col-8 col-medium-4">
```

File: templates/hpc/use-cases.html [hunk 2]
```diff
@@ -102,7 +102,31 @@ <h2 class="p-logo-section__title u-align--center">ADVANCING HIGH-PERFORMANCE WOR
...
+      <h2>Guide to high-performance computing</h2>
+      <p>Accelerate your high-performance computing projects with our guide. Learn all about HPC cluster deployment, schedulers and workloads.</p>
+      <p><a class="p-button--positive" href="https://ubuntu.com/engage/a-guide-to-high-performance-computing">Download our guide to HPC</a></p>
+    </div>
+    <div class="col-4 u-hide--small u-hide--medioum u-align--center u-vertically-center">
```
Comment (lyubomir-popov) [inline templates/hpc/use-cases.html:114]:
```suggestion
    <div class="col-4 col-medium-2 u-hide--small u-align--center u-vertically-center">
```

**Review by lyubomir-popov — APPROVED** [review]
I would keep the image on medium, suggestions in  the code. If you decide agianst it, please fix the typo (`u-hide--medioum` to be `u-hide--medium`)

**Review by mtruj013 — APPROVED** [review]
With the changes Lyubo requested for medium screens, LGTM!

**Pull request merged**